subscriber/podaac_data_subscriber.py

```diff
@@ -314,6 +314,13 @@ def run(args=None):
     logging.info("Downloaded Files: " + str(success_cnt))
     logging.info("Failed Files:     " + str(failure_cnt))
     logging.info("Skipped Files:    " + str(skip_cnt))
+
+    if success_cnt > 0:
+        try:
+            pa.create_citation_file(short_name, provider, data_path, token, args.verbose)
+        except:
+            logging.warning("Error generating citation")
```

Comment:
```suggestion
        except:
            logging.warning("Error generating citation", exc_info=True)
```
Is there a reason you went with warning here but debug in downloader?

Reply:
no...